recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: b28e72ac7a98be3d28ad28570299a393dfcd32e5e3f6a353dec94675767b6319
 
 build:
-  number: 2
-  skip: true  # [py2k]
+  number: 3
+  noarch: python
```

Comment:
There are no docs that describe when to change architecture to noarch but in team discussion was concluded to change it only with the new version and not with bumping build number

Reply:
Thanks!